src/store_digest.cc

```diff
@@ -421,7 +421,7 @@ storeDigestRewriteStart(void *)
     auto req = HttpRequest::FromUrlXXX(url, mx);
 
     RequestFlags flags;
-    flags.cachable = true;
+    flags.missCachingDecision = true;
```

Comment:
Setting this flag to true is a risky operation worth documenting why it needs to be true
```suggestion
    // XXX flag currently defaults to false. It should be true for ....
    flags.missCachingDecision = true; 
```